src/app/public/modules/colorpicker/colorpicker.component.html

```diff
@@ -19,20 +19,22 @@
     [attr.aria-label]="'skyux_colorpicker_dropdown_button' | skyLibResources"
     [attr.id]="triggerButtonId"
     [attr.title]="'skyux_colorpicker_dropdown_button' | skyLibResources"
-    [ngClass]="{
-      'sky-margin-inline-sm': themeName === 'modern'
```

Comment:
You can probably make `themeName` a private variable in the `.ts` file. Either that or get rid of it completely. It looks like we never use it outside of the `themeSvc.settingsChange.subscribe()` logic.